src/pizzeria/Pizza.java

```diff
@@ -0,0 +1,51 @@
+package pizzeria;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Pizza {
+    Order order = new Order();
+    private final List<String> pizzaName;
```

Comment:
pizzaName must be **String, not List**. So you have to change the logic for this class.
Also, you have to add field **pizzaType**, which can be ENUM, as you already did for **Ingredients**.
Also, you have to add field **List<Ingredients>**, because each pizza can have multiple ingredients.